src/common/reader.js

```diff
@@ -1197,8 +1199,37 @@ class Reader {
 						}
 						else {
 							this._lastView.navigate({ annotationID: annotation.id });
+							// When annotation is added from "Find in" popup, close it and re-focus the view
+							this.toggleFindPopup({ primary: this._lastViewPrimary, open: false });
+							this.focusView(this._lastViewPrimary);
```

Comment:
This is a broader issue, and even closing the find popup by clicking the close button doesn't restore focus to the view. However, we might want to keep it as is for now until I figure out the best way to resolve it.

Reply:
Understood, I removed that part. To give at least some indication that creating the annotation did work, I added one more announcement that will fire whenever an annotation is created (e.g. "Underline annotation has been created"). So, in this case, the popup will close and this message will be heard.